docs/src/content/docs/ru/guides/i18n.mdx

```diff
@@ -148,16 +143,42 @@ Starlight предполагает, что вы создадите эквива
 
 Если перевод для языка ещё не доступен, Starlight покажет читателям контент этой страницы на языке по умолчанию (установленном через `defaultLocale`). Например, если вы ещё не создали французскую версию вашей страницы «О нас» и вашим языком по умолчанию является английский, посетители `/fr/about` увидят английский контент из `/en/about` с уведомлением о том, что эта страница ещё не была переведена. Это помогает вам добавлять контент на вашем языке по умолчанию и постепенно переводить его, когда у ваших переводчиков есть время.
 
+## Перевод заголовка сайта
+
+По умолчанию Starlight использует одно и то же название сайта для всех языков.
+Если вам нужно настроить заголовок для каждой локали, вы можете передать объект [`title`](/ru/reference/configuration/#title-обязателен) в опциях Starlight:
+
+```diff lang="js"
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+
+export default defineConfig({
+  integrations: [
+    starlight({
+-     title: 'Моя документация',
++     title: {
++       en: 'My Docs',
++       ru: "Моя документация",
++     },
+      defaultLocale: 'ru',
+      locales: {
+        en: { label: 'English' },
+        ru: { label: 'Русский', lang: 'ru' },
```

Comment:
I think in this case, it would better to keep the `zh-CN` example and replace `en` by `ru`. This is what we've been doing so far in other translations and the goal is to keep an example with a BCP 47 tag with a regional subtag as they can be tricky with `zh-CN` vs `zh-cn`.